Directory.Build.targets

```diff
@@ -33,21 +29,10 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Coverage)' == 'true'">
-    <!-- https://github.com/tonerdo/coverlet/issues/363 -->
-    <DeterministicSourcePaths>false</DeterministicSourcePaths>
-
-    <!-- https://github.com/tonerdo/coverlet/issues/618 -->
-    <IncludeTestAssembly>true</IncludeTestAssembly>
-
-    <CollectCoverage>true</CollectCoverage>
-    <SingleHit>true</SingleHit>
-    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
-    <CoverletOutputFormat>opencover</CoverletOutputFormat>
-    <CoverletOutput>$(ArtifactsDir)coverage\$(MSBuildProjectName)_$(TargetFramework)_$(_TestArchitecture).coverage</CoverletOutput>
-    <Include></Include>
-    <Exclude></Exclude>
-    <ExcludeByAttribute>ExcludeFromCodeCoverage</ExcludeByAttribute>
-    <ExcludeByFile></ExcludeByFile>
+    <TestRunnerAdditionalArguments>--collect:"Code Coverage;Format=Cobertura"</TestRunnerAdditionalArguments>
+    <!-- TODO: On next arcade bump of Microsoft.CodeCoverage/Microsoft.NET.Test.Sdk we will have to update this line to be netstandard2.0 -->
+    <!-- Because arcade is passing dlls directly to 'dotnet test' command, we need to manually provide path to coverage datacollector -->
+    <TestRunnerAdditionalArguments>$(TestRunnerAdditionalArguments) --test-adapter-path $(NUGET_PACKAGES)/microsoft.codecoverage/$(MicrosoftNETTestSdkVersion)/build/netstandard1.0</TestRunnerAdditionalArguments>
```

Comment:
how do we know `microsoft.codecoverage` is in `NUGET_PACKAGES`? I understand that `microsoft.codecoverage` is a depndency of `Microsoft.NET.Test.Sdk` and arcade adds an implicit dependency to `Microsoft.NET.Test.Sdk`, but the `NUGET_PACKAGES` variable can be messed with in various ways. I would prefer adding `GeneratePathProperty="true"` to `microsoft.codecoverage` and then referencing its location via `$(PkgMicrosoft_CodeCoverage)/build/netstandard1.0`

Reply:
I wasn't aware of this `GeneratePathProperty` property. I will have a look into it and see if I can refactor the line above.

I am also linking the issue of arcade doing the bump of Test SDK + MS coverage: https://github.com/dotnet/arcade/issues/11856